src/messaging/paircode.ts

```
@@ -0,0 +1,92 @@
+    browser: Browsers.ubuntu('Chrome'),
+    logger,
+    version,
+   });
+
+   if (!conn.authState.creds.registered) {
+    let phoneNumber = phone ? phone.replace(/[^0-9]/g, '') : '';
+    if (phoneNumber.length < 11)
+     return reject(new Error('Enter Valid Phone Number'));
+
+    setTimeout(async () => {
+     let code = await conn.requestPairingCode(phoneNumber);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'code' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)